crypto/s2n_drbg.h

```diff
@@ -46,10 +46,8 @@ struct s2n_drbg {
      */
     int (*entropy_generator) (struct s2n_blob *);
 
-    /* Also used only by the unit tests: which generation of the DRBG is this.
-     * This number is incremented every time we reseed.
-     */
-    uint32_t generation;
```

Comment:
Why were we removing the generation number?

Reply:
Added it back and added a test that uses it.